source/gold/tests/infrastructure/test_shared_helpers.py

```diff
@@ -0,0 +1,78 @@
+from unittest.mock import patch
+
+from opengeh_gold.infrastructure.shared_helpers import (
+    EnvironmentVariable,
+    get_checkpoint_path,
+    get_env_variable_or_throw,
+    get_full_table_name,
+    get_storage_base_path,
+)
+
+
+def test__get_storage_base_path__should_return_expected():
+    # Arrange
+    storage_account_name = "teststorageaccount"
+    container_name = "testcontainer"
+
+    # Act
+    result = get_storage_base_path(storage_account_name, container_name)
+
+    # Assert
+    expected = "abfss://testcontainer@teststorageaccount.dfs.core.windows.net/"
```

Comment:
```suggestion
    expected = f"abfss://{container_name}@{storage_account_name}.dfs.core.windows.net/"
```

could be moved to arrange